ci/requirements-openvino-dev.in

```diff
@@ -0,0 +1,70 @@
+PyYAML (>=5.4.1)
```

Comment:
do we still need store this file in the repo if pip-compile already applied?

Reply:
With that it should be easier to track what openvino requires, so we won't need to reproduce it locally to figure out.

Reply:
ok